gateway/src/apicast/http_proxy.lua

```diff
@@ -96,10 +97,30 @@ local function forward_https_request(proxy_uri, uri)
         -- We cannot use resty.http's .get_client_body_reader().
         -- In POST requests with HTTPS, the result of that call is nil, and it
         -- results in a time-out.
+        --
+        --
+        -- If ngx.req.get_body_data is nil, can be that the body is too big to
+        -- read and need to be cached in a local file. This request will return
+        -- nil, so after this we need to read the temp file.
+        -- https://github.com/openresty/lua-nginx-module#ngxreqget_body_data
         body = ngx.req.get_body_data(),
         proxy_uri = proxy_uri
     }
 
+    if not request.body then
+        local temp_file_path = ngx.req.get_body_file()
+        ngx.log(ngx.INFO, "HTTPS Proxy: Request body is too big, read the content from path='", temp_file_path, "'")
```

Comment:
I think this could be a mere debug log level. Not fitting few kb does not really mean "too big".